wpilibc/src/main/native/cpp/DataLogManager.cpp

```diff
@@ -44,6 +44,7 @@ struct Thread final : public wpi::SafeThread {
 struct Instance {
   Instance(std::string_view dir, std::string_view filename, double period);
   wpi::SafeThreadOwner<Thread> owner;
+  ~Instance() { owner.GetThreadSharedPtr()->StopNTLog(); }
```

Comment:
The constructor and destructor should be grouped together.

Reply:
fixed